tests/integration.py

```diff
@@ -243,6 +245,40 @@ def test_custom_models(self):
     def tearDownClass(cls):
         prog_server.stop()
 
+    def test_custom_predictors(self):
+        # Restart server with model
+        prog_server.stop()
+        #define the model
+        ball = ThrownObject(thrower_height=1.5, throwing_speed=20)
+        prog_server.start(models={'ball': ball}, port=9883)
```

Comment:
This doesn't test the new feature. We need to pass the predictor into start function like we do with model here.

Then when we create a session we have to tell it to use the new predictor (which is actually just a Monte Carlo)